scraper/wgScraper.py

```diff
@@ -23,18 +25,27 @@
 db = client['plagiarismDB']
 windCol = db['windCollection']
 
+windCol.create_index("date", expireAfterSeconds=(2 * 86400))  # 2 days
+
 while True:
     browser.get(wgSite)
 
-    wait.until(ec.visibility_of_element_located((By.XPATH, "//span[@class='wgs_wind_avg_value']")))
+    wait.until(ec.visibility_of_element_located(
+        (By.XPATH, "//span[@class='wgs_wind_avg_value']")))
 
-    avg = float(browser.find_element_by_xpath("//span[@class='wgs_wind_avg_value']").text)
-    low = float(browser.find_element_by_xpath("//span[@class='wgs_wind_min_value']").text)
-    high = float(browser.find_element_by_xpath("//span[@class='wgs_wind_max_value']").text)
-    direction = browser.find_element_by_xpath("//span[@class='wgs_wind_dir_value']").text
+    avg = float(browser.find_element_by_xpath(
```

Comment:
Is this from a formatter?

Reply:
Indeed, you in favour or not?

Reply:
Good idea - will keep everything consistent